WarOfEmpires/Scripts/Application/grid.js

```diff
@@ -48,16 +48,30 @@
             cell.append($('<a>', { title: value || "", href: base.detailUrl + '?id=' + dataRow.Id }).text(value || ""));
```

Comment:
Please remove the || "" in two places in this row (and elsewhere in the document if applicable)